config_src/infra/FMS1/MOM_interp_infra.F90

```diff
@@ -14,7 +15,7 @@ module MOM_interp_infra
 
 implicit none ; private
 
-public :: horiz_interp_type, horiz_interp_init
+public :: horiz_interp_type, horiz_interp_init, horiz_interp_del
```

Comment:
I think that there is a consensus that we should not have simple pass-throughs of FMS routines, but rather we should have explicit interfaces for all of the routines (like horiz_interp_del) that are being made publicly visible, so that there is at least one place where all subroutine interfaces used in MOM6 are properly documented.  So the FMS routine `horiz_interp_del()` should be wrapped in another routine, perhaps `horiz_interp_delete()` that has comments describing the routine and documenting all of the arguments that we use.  I also note that some of the other interfaces in this file (like `horiz_interp_init()`) also are simple pass-throughs without adequate documentation.

Reply:
Fair point, but these are not isolated examples and there are other examples.

I also suspect that the whole operation could be reconsidered so that the MOM user does not need to explicit call any `_del()` operations.

I would suggest we accept these as following the established pattern, and address it in a future PR.

Reply:
Yes, upon further inspection of the code, I do see that there are a number of instances of undocumented pass-through interfaces from FMS. Especially given that these new interfaces are only used in modules in the src/framework code, within which there are well-documented interfaces, I think that upon further reflection I agree that these are OK.